scripts/build.sh

```diff
@@ -1,12 +1,58 @@
 #!/bin/bash
+# Single source of truth for building Firecloud-Ui.
+# @ Jackie Roberti and Isaac Zarsky
+#
+# Provide command line options to do one or several things:
+#   compile: compile clojure proj into /target
+#   -d | --docker : provide arg either "build" or "push", to build and push docker image
+# Jenkins build job should run with all options, for example,
+#   ./docker/build.sh compile -d push
```

Comment:
1. i suppose if you've already compiled the project, and just want to trigger the docker stuff.  i think it's good to have them separated.  the format is consistent across all the services, because some have different build steps that we may or may not want to do. 
2.  "-d" was to imply a docker command, bc we have multiple different ones supported
3.  help text would be good